classes/class-p4-campaigns.php

```diff
@@ -232,7 +232,7 @@ public function save_taxonomy_meta( $term_id ) {
 			}
 
 			$redirect_page = $_POST['redirect_page'] ?? 0;
-			if ( $this->validate_page_types( $redirect_page ) ) {
+			if ( get_post( $redirect_page ) ) {
```

Comment:
Looks good, but better check the return value

```suggestion
			if ( ! is_null( get_post( $redirect_page ) ) ) {
```